src/Tetris/PlayerSkeleton.java

```diff
@@ -1,14 +1,69 @@
 package Tetris;
 
+import java.util.*;
+
+import Tetris.Util.Tuple;
+import Tetris.Util.Util;
+
 public class PlayerSkeleton {
 
 	//implement this function to have a working system
 	public int pickMove(State s, int[][] legalMoves) {
-		
 		return 0;
 	}
-	
+
 	public static void main(String[] args) {
+		geneticFunction();
+	}
+
+	// Simulates the replacement of the population by its member's descendants
+	public static void geneticFunction() {
+		ArrayList<Heuristic> population = Util.getRandomHeuristics(Constants.NUMBER_OF_HEURISTICS);
```

Comment:
Can change the name of Util? It's slightly confusing as there's Java Util as well.

Reply:
Done